components/caselawmap.js

```diff
@@ -0,0 +1,92 @@
+import {LitElement, css, html} from '../lib/lit.js';
+import { baseStyles } from '../lib/wc-base.js';
+
+export class CaselawMap extends LitElement  {
+  static properties = {
+  };
+
+  static styles = [
+    baseStyles,
+    css`
+      :host {
+        color: var(--color-white);
+      }
+
+      .mapRegion {
+        background: var(--color-gradient);
+        padding-block: var(--spacing-325) var(--spacing-225);
```

Comment:
I'm using[ logical properties here](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values) because they don't make assumptions about what orientation a language is, eg, if it's a Left-to-right language.